### PR TITLE
update and tests against PHP 7.1

### PR DESCRIPTION
**Pull request by antograssiot**
It seems that the only modification required was linked to https://wiki.php.net/rfc/invalid_strings_in_arithmetic

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 language: php
 
 php:
+  - 7.1
```
Comment (ADmad) [inline .travis.yml]: I had kept 7.0 first so that coverage builds run first and report are send to codecov earlier :)

File: .travis.yml [hunk 2]
```diff
@@ -39,6 +40,7 @@ matrix:
 
   allow_failures:
     - php: hhvm
+    - php: 7.1
```
Comment (ADmad) [inline .travis.yml]: Since the tests are passing on 7.1 do we need to keep it under `allow_failures`?

File: .travis.yml (hunk 1 above)
Comment (antograssiot) [inline .travis.yml]: sure I'll update

File: .travis.yml (hunk 2 above)
Comment (antograssiot) [inline .travis.yml]: @ADmad there is one test on mysql which is always failling. I don't know the reason yet

Comment (antograssiot) [inline .travis.yml]: also it is still just beta. we can remove it then it will become RC

Comment (ADmad) [inline .travis.yml]: I doubt it's an "actual" failure. The php 5.6 job for mysql is also consistently failing. But nvm, we can always update later.

**Pull request merged**